client/data/onboarding/hooks.ts

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { STORE_NAME } from '../constants';
+
+export const useBusinessTypes = (): unknown =>
```

Comment:
Should add a return type here.

Reply:
For return type of hooks you should use the types of the returned object, in this case it should be:
```js
{
	businessTypes: Country[];
	isLoading: boolean;
} 
```

Reply:
Addressed in 0960b83e7e20849ec2ae6d19965aa716632ae6f8